test/core/credentials/transport/tls/spiffe_utils_test.cc

```diff
@@ -0,0 +1,145 @@
+//
+//
+// Copyright 2025 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+//
+
+#include "src/core/credentials/transport/tls/spiffe_utils.h"
+
+#include <grpc/grpc.h>
+
+#include <string>
+
+#include "absl/status/status.h"
+#include "absl/status/statusor.h"
+#include "absl/strings/str_cat.h"
+#include "absl/strings/str_split.h"
+#include "absl/strings/string_view.h"
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+#include "test/core/test_util/test_config.h"
+
+using ::grpc_core::experimental::SpiffeId;
```

Comment:
This shouldn't be needed once you remove the `experimental` namespace, as I suggested elsewhere.

Reply:
Removed